skin/adminhtml/default/default/boxes.css

```diff
@@ -102,7 +102,7 @@
     z-index:1000;
     }
 
-#message-popup-window-mask { position:absolute; top:0; right:0; bottom:0; left:0; width:100%; height:100%; z-index:1980; background-color:#efefef; opacity:.5; }
+#message-popup-window-mask { position:absolute; top:0; right:0; bottom:0; left:0; width:100%; height:100%; z-index:1980; background-color:#efefef; opacity:.5; -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=50)";/*IE8*/ }
```

Comment:
Didn't we remove IE support from adminhtml already?  Or do I misinterpret ` -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=50)";/*IE8*/ ` here?

Reply:
OMG this is my nightmare, you're right, thank you very much. I fixed it.

Reply:
We ditched IE for a while. I would recommend a comparison between this modified file and the one released in 20.0.14.